CMake/AddCLI11Submodule.cmake

```diff
@@ -17,12 +17,12 @@ find_package_handle_standard_args(CLI11 REQUIRED_VARS CLI11_PROJ)
 if(NOT CLI11_FOUND)
   find_package(Git 1.8.3 QUIET)
   if(NOT ${GIT_FOUND})
-    message(FATAL_ERROR "git not found, clone repository with --recursive")
+    message(FATAL_ERROR "git not found, clone repository with")
```

Comment:
Don't understand why the message ends with 'with'. Do you mean "git not found. Please install git."

Reply:
Ah yes, thats an error!